pkg/dao/repository_test.go

```diff
@@ -256,24 +272,44 @@ func (suite *RepositorySuite) TestUpdateEmpty() {
 	name := "Updated"
 	arch := ""
 	t := suite.T()
+	tx := suite.tx
 	org_id := seeds.RandomOrgId()
 	var err error
 
-	err = seeds.SeedRepositoryConfigurations(suite.tx, 1,
-		seeds.SeedOptions{OrgID: org_id, Arch: pointy.String(config.X8664)})
-	assert.Nil(t, err)
-	found := models.RepositoryConfiguration{}
-	suite.tx.Where("org_id = ?", org_id).First(&found)
+	// Create a RepositoryConfiguration record
+	repoPublic := repoPublicTest.DeepCopy()
+	err = tx.Create(&repoPublic).Error
+	require.NoError(t, err)
 
+	repoConfig := repoConfigTest1.DeepCopy()
+	repoConfig.RepositoryUUID = repoPublic.UUID
+	repoConfig.OrgID = org_id
+	err = tx.Create(&repoConfig).Error
```

Comment:
I add the records without SeedRepositoryConfigurations to have control about the uuid to check later the records.